doc/source/tutorials.rst

```diff
@@ -0,0 +1,256 @@
+
+To use BraketClientBackend, we import these packages first.
+
+.. code-block:: python
+
+   from braket.aws import AwsDevice
+   from braket.devices import Devices, LocalSimulator
+   from qibo_cloud_backend.braket_client import BraketClientBackend
+
+Then, construct a Qibo circuit.
+
+.. code-block:: python
+
+   from qibo import gates, Circuit as QiboCircuit
+   import numpy as np
+
+   c = QiboCircuit(2)
+   c.add(gates.H(0))
+   c.add(gates.CNOT(0, 1))
+   c.add(gates.M(0))
+   c.add(gates.M(1))
+
+   print(c.draw())
+
+We should get this circuit:
+
+.. code-block:: python
+
+   q0: -H-o-M-
+   q1: ---x-M-
+
+Now, we initialize the Amazon Braket device and execute circuit `c` on the backend `AWS`.
+
+.. code-block:: python
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   AWS = BraketClientBackend(device = device, verbatim_circuit=False)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+For completeness, one can also use the LocalSimulator to execute circuit `c` as follows.
+
+.. code-block:: python
+
+   device = LocalSimulator("default")
+   AWS = BraketClientBackend(device = device, verbatim_circuit=False)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+
+Submit a circuit in verbatim mode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In verbatim mode, the circuit is executed on the device without any transpilation. The user has to ensure that the circuit is specifically written in the device's native gates and gates obey the topology of the device.
+Therefore, before submitting a Qibo circuit in verbatim mode, it is recommended to extract the Amazon Braket device's information. We will demonstrate this below.
+
+Extracting Amazon Braket device parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+More information on IQM Garnet device can be found here: https://aws.amazon.com/braket/quantum-computers/iqm/.
+
+To use BraketClientBackend, we import these packages first.
+
+.. code-block:: python
+
+   from braket.aws import AwsDevice
+   from braket.devices import Devices, LocalSimulator
+   from qibo_cloud_backend.braket_client import BraketClientBackend
+
+The qubit connectivity on the IQM Garnet device can be visualised using networkx. We will also extract the device's native gates.
+
+.. code-block:: python
+
+   import networkx as nx
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   connectivity_graph = AwsDevice(device).properties.paradigm.connectivity.connectivityGraph
+   native_gates = AwsDevice(device).properties.paradigm.nativeGateSet
+   print(native_gates)
+
+   G = nx.Graph()
+   for node, neighbors in connectivity_graph.items():
+      for neighbor in neighbors:
+         G.add_edge(node, neighbor)
+
+   nx.draw(G, pos=nx.spring_layout(G), with_labels=True, node_color='lightblue', node_size=500, font_size=10, font_weight='bold', edge_color='gray')
```

Comment:
I feel that it would be useful to leave it here as it's a tutorial about using BraketClientBackend. I'm afraid that links to the API reference might have too much information for the user to extract quickly. I have shortened this segment to the following:

```
Extracting Amazon Braket device parameters
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The Amazon Braket devices can be found here: https://aws.amazon.com/braket/quantum-computers/.
Using the `IQM Garnet device <https://aws.amazon.com/braket/quantum-computers/iqm/>`_ as an example,
we demonstrate how to extract useful device information such as the qubit topology and native gates.
The qubit connectivity on the IQM Garnet device can be visualised using `NetworkX <https://networkx.org/>`_.

.. code-block:: python

   import networkx as nx

   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
   connectivity_graph = AwsDevice(device).properties.paradigm.connectivity.connectivityGraph
   native_gates = AwsDevice(device).properties.paradigm.nativeGateSet
   print(native_gates)

   G = nx.Graph()
   for node, neighbors in connectivity_graph.items():
      for neighbor in neighbors:
         G.add_edge(node, neighbor)

   nx.draw(G, pos=nx.spring_layout(G), with_labels=True, node_color='lightblue', 
           node_size=500, font_size=10, font_weight='bold', edge_color='gray')
```

Let me know what you think!

Reply:
Nice